test-clients/src/main/java/com/hedera/services/bdd/suites/reconnect/ValidateCongestionPricingAfterReconnect.java

```diff
@@ -0,0 +1,215 @@
+import static com.hedera.services.bdd.suites.reconnect.ValidateTokensStateAfterReconnect.reconnectingNode;
+import static com.hedera.services.bdd.suites.utils.sysfiles.serdes.ThrottleDefsLoader.protoDefsFromResource;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.ACCOUNT_IS_TREASURY;
+
+/**
+ * A reconnect test in which a congestion pricing multiplier is updated and triggered while the node 0.0.8 is
+ * disconnected from the network. Once the node is reconnected validate that the congestion pricing is in affect on
+ * reconnected node
+ */
+public class ValidateCongestionPricingAfterReconnect extends HapiApiSuite {
+	private static final Logger log = LogManager.getLogger(ValidateCongestionPricingAfterReconnect.class);
+	private static final String defaultCongestionMultipliers =
+			HapiSpecSetup.getDefaultNodeProps().get("fees.percentCongestionMultipliers");
+	private static final String defaultMinCongestionPeriod =
+			HapiSpecSetup.getDefaultNodeProps().get("fees.minCongestionPeriod");
+
+	public static void main(String... args) {
+		new ValidateAppPropertiesStateAfterReconnect().runSuiteSync();
+	}
+
+	@Override
+	protected List<HapiApiSpec> getSpecsInSuite() {
+		return List.of(
+				new HapiApiSpec[] {
+						validateCongestionPricing(),
+				}
+		);
+	}
+
+	private HapiApiSpec validateCongestionPricing() {
+		var artificialLimits = protoDefsFromResource("testSystemFiles/artificial-limits-6N.json");
+		var defaultThrottles = protoDefsFromResource("testSystemFiles/throttles-dev.json");
+		String tmpMinCongestionPeriodInSecs = "5";
+		String civilianAccount = "civilian";
+		String oneContract = "contract";
+
+		AtomicLong normalPrice = new AtomicLong();
+		AtomicLong tenXPrice = new AtomicLong();
+
+		return customHapiSpec("ValidateCongestionPricing")
+				.withProperties(Map.of(
+						"txn.start.offset.secs", "-5")
+				)
+				.given(
+						sleepFor(Duration.ofSeconds(25).toMillis()),
+
+						cryptoCreate(civilianAccount)
+								.payingWith(GENESIS)
+								.balance(ONE_MILLION_HBARS)
+								.logging(),
+						fileCreate("bytecode")
+								.path(ContractResources.MULTIPURPOSE_BYTECODE_PATH)
+								.payingWith(GENESIS).logging(),
+						contractCreate(oneContract)
+								.bytecode("bytecode").logging()
+								.payingWith(GENESIS),
+						contractCall(oneContract)
+								.payingWith(civilianAccount)
+								.fee(ONE_HUNDRED_HBARS)
+								.sending(ONE_HBAR)
+								.via("cheapCallBeforeCongestionPricing").logging(),
+						getTxnRecord("cheapCallBeforeCongestionPricing")
+								.providingFeeTo(normalPrice::set).logging(),
+						getAccountBalance(GENESIS)
+								.setNode(reconnectingNode)
+								.unavailableNode().logging()
+				).when(
+						/* update the multiplier to 10x with a 1% congestion for tmpMinCongestionPeriodInSecs */
+						fileUpdate(APP_PROPERTIES)
+								.fee(ONE_HUNDRED_HBARS)
+								.payingWith(EXCHANGE_RATE_CONTROL)
```

Comment:
can we use Admin account 0.0.50 instead of 0.0.57 `EXCHANGE_RATE_CONTROL` to update Application properties

Reply:
@anighanta  Based on this [matrix](https://github.com/hashgraph/hedera-services/blob/master/docs/privileged-transactions.md#authorization-privileges-for-file-updates-and-appends), account 2, 50, 55, and 57 have privileges to change the file. Is there any particular reason why we should only use account 50 ?